arm/ui/settings/templates/settings/ui.html

```diff
@@ -3,113 +3,14 @@
-                       name="bootstrap_skin" placeholder="" value="{{ ui_settings.bootstrap_skin }}"
-                       aria-describedby="bootstrap_skin">
-                <a class="popovers" onClick='return false;' href=""
-                   data-content="{{ jsoncomments['bootstrap_skin']| replace("#", "\n") }}"
-                   rel="popover"
-                   data-placement="top" data-original-title="bootstrap_skin">
-                    <img title="More information" src="{{ url_for('static', filename='img/info.png') }}" width="30px"
-                         height="35px" alt="More info">
-                </a>
-            </div>
-
-            <div class="input-group mb-3">
-                <div class="input-group-prepend">
-                    <span class="input-group-text" id="language">language: </span>
-                </div>
-                <input type="text" class="form-control" aria-label="language" name="language"
-                       placeholder="" value="{{ ui_settings.language }}"
-                       aria-describedby="language">
-                <a class="popovers" onClick='return false;' href=""
-                   data-content="{{ jsoncomments['language']| replace("#", "\n") }}" rel="popover"
-                   data-placement="top" data-original-title="language">
-                    <img title="More information" src="{{ url_for('static', filename='img/info.png') }}" width="30px"
-                         height="35px" alt="More info">
-                </a>
-            </div>
-
-            <div class="input-group mb-3">
-                <div class="input-group-prepend">
-                    <span class="input-group-text" id="database_limit">database_limit: </span>
-                </div>
-                <input type="text" class="form-control" aria-label="database_limit"
-                       name="database_limit" placeholder="" value="{{ ui_settings.database_limit }}"
-                       aria-describedby="database_limit">
-                <a class="popovers" onClick='return false;' href=""
-                   data-content="{{ jsoncomments['database_limit']| replace("#", "\n") }}"
-                   rel="popover"
-                   data-placement="top" data-original-title="database_limit">
-                    <img title="More information" src="{{ url_for('static', filename='img/info.png') }}" width="30px"
-                         height="35px" alt="More info">
-                </a>
-            </div>
+            {% from "settings/_form_field.html" import render_field %}
+            {% for field in ui_settings %}
+                {% if field.name != 'submit' %}
+                    {% if field.name != 'csrf_token' %}
+                    {{ render_field(field) }}
+                    {% endif %}
+                {% endif %}
+            {% endfor %}
             <button id="uiSubmit" class="btn btn-secondary btn-lg btn-block" form="uiSettings"
```

Comment:
This no longer submits when the return/enter key is pressed.

Also, the layout is broken and will need the CSS looked at, as all the inputs are not correctly evenly sized

Reply:
The functionality to save on enter has not been removed, it's the main method I have been using to test this (CBF scrolling all the way down).

You might find that the arm.yaml your testing against is throwing form based validation errors. 
<img width="1867" height="261" alt="settings_form_error" src="https://github.com/user-attachments/assets/53587cac-6d92-4d88-84a6-f05182ea43f5" />

Those are form validation errors, for the 'template' arm.yaml.

Reply:
Is there an issue with the template or the new parsing of the arm.yaml template/default?